src/Sarif.FunctionalTests/v2/SpecExamples/Comprehensive.sarif

```diff
@@ -20,6 +20,15 @@
           "copyright": "Copyright (c) 2016 by Example Corporation. All rights reserved."
         }
       },
+      "versionControlProvenance": [
+        {
+          "uri": "https://github.com/contoso/example",
```

Comment:
>github [](start = 26, length = 6)

`example.com`

Reply:
microsoft has created a contoso organization on github.com, do you see an issue using it? 

---
In reply to: [221319843](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221319843) [](ancestors = 221319843)

Reply:
I didn't know that.

---
In reply to: [221400882](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221400882) [](ancestors = 221400882,221319843)